numpy/core/machar.py

```diff
@@ -316,6 +324,8 @@ def _do_init(self, float_conv, int_conv, float_to_float, float_to_str, title):
         self.epsilon = self.eps
         self.tiny = self.xmin
         self.huge = self.xmax
+        self.smallest_normal = self.xmin
```

Comment:
If it's just an alias anyway, might it not make more sense to define it as a property that wraps around `finfo.xmin`?

Reply:
We could do that as well, @seberg will we want to go for this approach?

Reply:
I honestly don't care.